broker/src/test/java/org/apache/rocketmq/broker/topic/TopicQueueMappingManagerTest.java

```diff
@@ -44,12 +44,12 @@
 public class TopicQueueMappingManagerTest {
     @Mock
     private BrokerController brokerController;
-    private static final String broker1Name = "broker1";
+    private static final String BROKER1_NAME = "broker1";
```

Comment:
```suggestion
    private static final String BROKER_NAME = "broker1";
```
IMO, it could be defined as a brokerName directly because there is only one test broker

Reply:
IMO, if defined as brokerName, it should not be a constant. Both are ok.

Reply:
Let's keep this pull request as focused as possible. Keep the code as it is and change as minimal as possible to make checkstyle pass.